lambdas/tests/unit/services/test_bulk_upload_service.py

```diff
@@ -190,6 +190,7 @@ def test_lambda_handler_handle_pds_too_many_requests_exception(
         mock_back_to_queue.assert_any_call(message)
 
 
+@pytest.mark.focus
```

Comment:
What is this? I've taken a look at this on google and it looks like it forces pytest to run this test exclusively, if I've understood correctly, not something we'd want to commit